CMakePresets.json

```diff
@@ -0,0 +1,528 @@
+        "CMAKE_CXX_COMPILER": "cl"
+      }
+    },
+    {
+      "name": "windows-msvc-debug-ootree-fast",
+      "displayName": "Windows x64: MSVC / Debug / Out of tree (fast)",
+      "inherits": "windows-config-base",
+      "binaryDir": "${sourceDir}/.build/${presetName}",
+      "installDir": "${sourceDir}/install",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
+        "RE2C_BUILD_RE2GO": false
+      }
+    },
+    {
+      "name": "windows-msvc-debug-ootree-full",
+      "displayName": "Windows x64: MSVC / Debug / Out of tree (full)",
+      "inherits": "windows-msvc-debug-ootree-fast",
+      "installDir": "${sourceDir}/install-2",
+      "cacheVariables": {
+        "RE2C_BUILD_RE2GO": true,
+        "RE2C_BUILD_LIBS": true,
+        "RE2C_REBUILD_LEXERS": true,
+        "RE2C_REBUILD_DOCS": false,
+        "RE2C_FOR_BUILD": "${sourceDir}/install/bin/re2c"
+      }
+    },
+    {
+      "name": "windows-msvc-debug-intree-fast",
+      "displayName": "Windows x64: MSVC / Debug / In tree (fast)",
+      "inherits": "windows-config-base",
+      "binaryDir": "${sourceDir}",
+      "installDir": "${sourceDir}/install",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
+        "RE2C_BUILD_RE2GO": false
+      }
+    },
+    {
+      "name": "windows-msvc-debug-intree-full",
+      "displayName": "Windows x64: MSVC / Debug / In tree (full)",
+      "inherits": "windows-msvc-debug-intree-fast",
+      "installDir": "${sourceDir}/install-2",
+      "cacheVariables": {
+        "RE2C_BUILD_RE2GO": true,
+        "RE2C_BUILD_LIBS": true,
+        "RE2C_REBUILD_LEXERS": true,
+        "RE2C_REBUILD_DOCS": false,
+        "RE2C_FOR_BUILD": "${sourceDir}/install/bin/re2c"
+      }
+    },
+    {
+      "name": "windows-msvc-release-ootree-fast",
+      "displayName": "Windows x64: MSVC / Release / Out of tree (fast)",
+      "inherits": "windows-config-base",
+      "binaryDir": "${sourceDir}/.build/${presetName}",
+      "installDir": "${sourceDir}/install",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Release",
+        "RE2C_BUILD_RE2GO": false
+      }
+    },
+    {
+      "name": "windows-msvc-release-ootree-full",
+      "displayName": "Windows x64: MSVC / Release / Out of tree (full)",
+      "inherits": "windows-msvc-release-ootree-fast",
+      "installDir": "${sourceDir}/install-2",
+      "cacheVariables": {
+        "RE2C_BUILD_RE2GO": true,
+        "RE2C_BUILD_LIBS": true,
+        "RE2C_REBUILD_LEXERS": true,
+        "RE2C_REBUILD_DOCS": false,
+        "RE2C_FOR_BUILD": "${sourceDir}/install/bin/re2c"
+      }
+    }
+  ],
+  "buildPresets": [
+    {
+      "name": "linux-build-base",
+      "hidden": true,
+      "configuration": "Debug",
+      "condition": {
+        "type": "equals",
+        "lhs": "${hostSystemName}",
+        "rhs": "Linux"
+      }
+    },
+    {
+      "name": "linux-gcc-ubsan-ootree-fast",
+      "displayName": "Linux x64: GCC / Ubsan / Out of tree (fast)",
```

Comment:
Maybe x86_64 (here and in other places)?

Reply:
Please take a look: https://github.com/skvadrik/re2c/pull/374/commits/216932105275c3017cf0bf8bbb05c2492d09fa3d